### ISSUE TITLE
Why use padding = [1, 1] in FBNet V3 ?

### ISSUE THREAD

**Issue by changlin31** [facebookresearch/mobile-vision#53]
In FBNet V3, the padding of downsampling layer is set to [1, 1], resulting in smaller resolution in later stages (i.e. 13x13 and 6x6 in the last two stages). This is not a common setting in other network, including MBNet and previous version of FBNet. 

Why do you make such adjustment in FBNet V3? How does it affect the accuracy? Thanks

**Comment by zhanghang1989** [comment]
The architecture is searched automatically.

**Comment by changlin31** [comment]
I know, but I believe padding is not included in the search space.

**Comment by laisimiao** [comment]
Does this repo provide model or codes of v3 version? @changlin31

**Comment by changlin31** [comment]
Yes, @laisimiao  here: https://github.com/facebookresearch/mobile-vision/blob/b2964a4476167589b0f7fad207bf51f614af5fd7/mobile_cv/arch/fbnet_v2/fbnet_modeldef_cls_fbnetv3.py

**Comment by laisimiao** [comment]
@changlin31  Thanks. But it just provide the pretrained model with v1 and v2?

**Comment by sbl1996** [comment]
@changlin31 Do you know the reason of the uncommon padding now? Thanks very much.

**Comment by changlin31** [comment]
@sbl1996 I have no clue. Maybe it's just a trick to reduce the FLOPS.
